schema/ListedLicense.xsd

```diff
@@ -80,14 +272,54 @@
 	</group>
 	<group name="formattedAltTextGroup">
 		<choice>
-			<element name="p" type="tns:formattedAltTextType" minOccurs="0" maxOccurs="unbounded"/>
-			<element name="bullet" type="string" minOccurs="0" maxOccurs="unbounded"/>
+			<element name="p" type="tns:formattedAltTextType" minOccurs="0" maxOccurs="unbounded">
+				<annotation>
+					<documentation xml:lang="en">
+						<xhtml:p>
+							<xhtml:code>p</xhtml:code> is a paragraph with the same semantics as in <xhtml:a href="https://www.w3.org/TR/html52/grouping-content.html#elementdef-p">HTML</xhtml:a>.
+						</xhtml:p>
+					</documentation>
+				</annotation>
+			</element>
+			<element name="bullet" type="string" minOccurs="0" maxOccurs="unbounded">
+				<annotation>
+					<documentation xml:lang="en">
+						<xhtml:p>
+							<xhtml:code>bullet</xhtml:code> is a list-item bullet.
+						</xhtml:p>
+					</documentation>
+				</annotation>
+			</element>
 			<element name="list" type="tns:listType" minOccurs="0" maxOccurs="unbounded"/>
 			<element name="optional" type="tns:optionalType" minOccurs="0" maxOccurs="unbounded"/>
 			<element name="alt" type="tns:altType" minOccurs="0" maxOccurs="unbounded"/>
-			<element name="br" type="tns:emptyType" minOccurs="0" maxOccurs="unbounded"/>
-			<element name="titleText" type="tns:formattedAltTextType" minOccurs="0" maxOccurs="1"/>
-			<element name="copyrightText" type="tns:formattedAltTextType" minOccurs="0" maxOccurs="1"/>
+			<element name="br" type="tns:emptyType" minOccurs="0" maxOccurs="unbounded">
+				<annotation>
+					<documentation xml:lang="en">
+						<xhtml:p>
+							<xhtml:code>br</xhtml:code> is a line break with the same semantics as in <xhtml:a href="https://www.w3.org/TR/html52/textlevel-semantics.html#the-br-element">HTML</xhtml:a>.
+						</xhtml:p>
+					</documentation>
+				</annotation>
+			</element>
+			<element name="titleText" type="tns:formattedAltTextType" minOccurs="0" maxOccurs="1">
+				<annotation>
+					<documentation xml:lang="en">
+						<xhtml:p>
+							<xhtml:code>titleText</xhtml:code> is the title of the license/exception.
```

Comment:
Do we want to add the titleText is always considered optional per the license matching guidelines?

Reply:
> Do we want to add the `titleText` is always considered optional per the license matching guidelines?

I don't think we need to call that out, since I'd rather move towards having distinct markup for optionality (`<alt>` and `<optional>`) and semantic meaning (e.g. `<h1>`).  It's my understanding that spdx/tools doesn't treat `<titleText>` as optional at the moment, is that the case?

Reply:
I just checked and the tools to convert the title to optional

Reply:
Ok, I've pushed deec602 → 48cf583 which mentions both the title-ness and optional-ness.  I've filed #607 to track my proposed change.